compiler/damlc/daml-compiler/src/DA/Daml/Compiler/DataDependencies.hs

```diff
@@ -613,10 +613,9 @@ generateSrcFromLf env = noLoc mod
     interfaceDecls :: [Gen (LHsDecl GhcPs)]
     interfaceDecls = do
         interface <- NM.toList $ LF.moduleInterfaces $ envMod env
-        [interfaceName] <- [LF.unTypeConName $ LF.intName interface]
-        let interfaceType = HsTyVar noExt NotPromoted $ mkRdrName interfaceName
         meth <- NM.toList $ LF.intMethods interface
         pure $ do
+            interfaceType <- convType env reexportedClasses . LF.TCon . qualify . LF.intName $ interface
```

Comment:
damnit, I was wrong with guessing that it would be a one line change. It deleted two lines. Close enough though :)

Reply:
-1 line change ftw :D